basic.tmuxtheme

```diff
@@ -22,12 +19,10 @@ set -g window-status-format " #I:#W#F "
 set -g window-status-current-format " #I:#W#F "
 
 # Current window status
-set -g window-status-current-bg red
-set -g window-status-current-fg black
+set -g window-status-current-style bg=red,fg=black
 
 # Window with activity status
-set -g window-status-activity-bg yellow # fg and bg are flipped here due to a
-set -g window-status-activity-fg black  # bug in tmux
+set -g window-status-activity-style bg=yellow,fg=black # fg and bg are flipped here due to a bug in tmux
```

Comment:
It seems the `-style` option doesn't have the flipped bug, so could you flip the values around? :)

Reply:
sure, addressed in https://github.com/jimeh/tmux-themepack/pull/25/commits/7bb2aa5c1c8bca6f3f1799443873c35f14433e51